ts/features/itwallet/machine/credential/context.ts

```diff
@@ -3,13 +3,15 @@ import { StoredCredential } from "../../common/utils/itwTypesUtils";
 
 export type Context = {
   credentialType: CredentialType | undefined;
+  hardwareKeyTag: string | undefined;
```

Comment:
Do we need this change?

Reply:
Nope, this is a merge artifact. It keeps popping up in every PR.....die hard 🥲

Reply:
Addressed here: https://github.com/pagopa/io-app/pull/5948/commits/2c3dd28fb6676ff00e16e05997d6e232aa232999